### ISSUE TITLE
Disable all rules by default

### ISSUE THREAD

**Issue by lencioni** [brigade/scss-lint#564]
I think that as scss-lint grows, it would be least surprising and easiest to understand for all rules to be disabled by default.

Perhaps this change could be coupled with a 1.0 release, which is [the route that ESLint recently took](http://eslint.org/blog/2015/07/eslint-1.0.0-released/#reset-is-now-the-default). Migrating should be pretty painless for folks, especially if we provide a migration guide (such as [the migration guide that the ESLint team provided for their analogous change](http://eslint.org/docs/user-guide/migrating-to-1.0.0#all-rules-off-by-default)).

It might be nice to couple this with a feature that would allow people to package up and distribute their scss-lint configuration, which would allow us to release a recommended configuration at the same time. ESLint also has this feature, and one of the more well-known uses of this is the [Airbnb ESLint config](https://www.npmjs.com/package/eslint-config-airbnb) which is part of [their JavaScript style guide](https://github.com/airbnb/javascript).

Thoughts?

**Comment by sds** [comment]
Agreed that we should have all linters disabled by default for the 1.0.0 release. 

Until then, any new linters will be automatically disabled going forward.

**Comment by connorshea** [comment]
:+1: agreed, would like this change. Any idea of when 1.0.0 will be released?
